project1-issue34/README.md

```diff
@@ -1,6 +1,32 @@
-Update this README to include the following:
-- A brief description of the project
-- Link to your issue & PR
-- A brief description of the changes you made
-- A brief description of the challenges you faced
-- Any other relevant information
+# Project Tirtha - Winter of Code 2024
+
+## Brief Description
+
+-This contribution was part of Winter of Code 2024 and I was working on this Project Tritha.
```

Comment:
Remove the `-` and reword the first line. It doesn't sound correct semantically.